### PR TITLE
get rid of anonymous imports

### PR DESCRIPTION
**Pull request by zramsay**
clearer code FTW

### CODE REVIEW

File: p2p/upnp/README.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# `tendermint/p2p/upnp`
-
-## Resources
-
-* http://www.upnp-hacks.org/upnp.html
```
Comment (ebuchman) [inline p2p/upnp/README.md]: lets keep this link somewhere!

File: rpc/client/rpc_test.go [hunk 2]
```diff
@@ -6,7 +6,8 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/tendermint/merkleeyes/iavl"
+
+	"github.com/tendermint/merkleeyes/iavl" //TODO use tendermint/iavl ?
```
Comment (ebuchman) [inline rpc/client/rpc_test.go]: yes please. will take a bit of fanagling

Comment (zramsay) [reply rpc/client/rpc_test.go]: #713

File: p2p/upnp/README.md (hunk 1 above)
Comment (zramsay) [reply p2p/upnp/README.md]: see top of upnp.go

**Pull request merged**